### ISSUE TITLE
Does not support MSSQL

### ISSUE THREAD

**Issue by tmcdonnell87** [llambda/connect-session-knex#36]
Still one of the most popular DB choices, unfortunately. 

Requests generate the following:
```
Unhandled rejection RequestError: Arithmetic overflow error converting expression to data type datetime.
    at /mnt/c/Users/Terry/Projects/gohealth/node_modules/mssql/lib/tedious.js:739:17
    at emitOne (events.js:96:13)
    at Connection.emit (events.js:191:7)
    at Parser.<anonymous> (/mnt/c/Users/Terry/Projects/gohealth/node_modules/tedious/lib/connection.js:204:16)
    at emitOne (events.js:96:13)
    at Parser.emit (events.js:191:7)
    at Parser.<anonymous> (/mnt/c/Users/Terry/Projects/gohealth/node_modules/tedious/lib/token/token-stream-parser.js:42:15)
    at emitOne (events.js:96:13)
    at Parser.emit (events.js:191:7)
    at readableAddChunk (/mnt/c/Users/Terry/Projects/gohealth/node_modules/tedious/node_modules/readable-stream/lib/_stream_readable.js:212:18)
    at Parser.Readable.push (/mnt/c/Users/Terry/Projects/gohealth/node_modules/tedious/node_modules/readable-stream/lib/_stream_readable.js:171:10)
    at Parser.Transform.push (/mnt/c/Users/Terry/Projects/gohealth/node_modules/tedious/node_modules/readable-stream/lib/_stream_transform.js:123:32)
    at doneParsing (/mnt/c/Users/Terry/Projects/gohealth/node_modules/tedious/lib/token/stream-parser.js:87:17)
    at /mnt/c/Users/Terry/Projects/gohealth/node_modules/tedious/lib/token/infoerror-token-parser.js:46:5
    at /mnt/c/Users/Terry/Projects/gohealth/node_modules/tedious/lib/token/infoerror-token-parser.js:13:19
    at /mnt/c/Users/Terry/Projects/gohealth/node_modules/tedious/lib/token/stream-parser.js:224:9
```

**Comment by bdwade100** [comment]
I added MSSQL support on my fork and added pull request #37.

**Comment by llambda** [comment]
Fixed by #37
